Source/WebCore/platform/mediastream/mac/CoreAudioSharedUnit.cpp

```diff
@@ -681,8 +682,24 @@ bool CoreAudioSharedUnit::migrateToNewDefaultDevice(const CaptureDevice& capture
     handleNewCurrentMicrophoneDevice(WTFMove(*device));
     return true;
 }
-#endif
 
+void CoreAudioSharedUnit::prewarmAudioUnitCreation(CompletionHandler<void()>&& callback)
+{
+    if (!m_audioUnitCreationWarmupPromise) {
+        m_audioUnitCreationWarmupPromise = invokeAsync(WorkQueue::create("CoreAudioSharedUnit AudioUnit creation").get(), [] {
+            return createAudioUnit(true);
+        })->whenSettled(RunLoop::main(), [weakThis = WeakPtr { *this }] (auto&& vpioUnitOrError) {
+            if (weakThis && vpioUnitOrError.has_value())
+                weakThis->setStoredVPIOUnit(WTFMove(vpioUnitOrError.value()));
+            return WTF::GenericNonExclusivePromise::createAndResolve();
```

Comment:
WTF:: isn't needed.

Reply:
Should we handle errors ?

Reply:
No, this is prewarming. If it fails, we will just retry when actually starting to capture and fail capture if it fails again.